lang/简体中文.json

```diff
@@ -1,47 +1,47 @@
 {
     "english_name": "Simplified Chinese",
     "status": {
-        "terminated": "\n应用程序已终止，请关闭窗口以退出应用程序.",
-        "watching": "正在观看: {channel}",
-        "goes_online": "{channel} 该频道已上线, 正在切换...",
-        "goes_offline": "{channel} 该频道已离线, 正在切换...",
-        "claimed_drop": "已领取掉宝: {drop}",
-        "no_channel": "没有可观看的频道，等待一个在线直播频道...",
-        "no_campaign": "没有可用于掉宝的活动，等待一个有效的掉宝活动..."
+        "terminated": "\n应用程序已停止运行，请关闭此窗口以退出应用程序。",
+        "watching": "正在观看：{channel}",
+        "goes_online": "“{channel}”频道已上线, 正在切换…",
+        "goes_offline": "“{channel}”频道已离线, 正在切换…",
+        "claimed_drop": "已领取掉宝：“{drop}”",
+        "no_channel": "没有可观看的频道，正在等待直播频道上线…",
+        "no_campaign": "没有可用于掉宝的活动，正在等待掉宝活动开放…"
     },
     "login": {
-        "unexpected_content": "返回的内容类型有误通常是由于被重定向.您需要VPN才能登录吗？ ",
+        "unexpected_content": "返回的内容类型有误通常是由于被重定向。您是否需要先设置VPN或登录到当前网络？",
         "chrome": {
-            "startup": "正在打开Chrome浏览器...",
-            "login_to_complete": "请再次点击登录按钮来完成登录.",
-            "no_token": "无法获取授权令牌.",
-            "closed_window": "在登录完成之前关闭了Chrome浏览器."
+            "startup": "正在打开Chrome浏览器…",
+            "login_to_complete": "请再次点击登录按钮来完成登录。",
+            "no_token": "无法获取授权令牌。",
+            "closed_window": "Chrome浏览器窗口在登录完成前被关闭。"
         },
         "error_code": "登录出错: {error_code}",
-        "incorrect_login_pass": "用户名或密码错误.",
-        "incorrect_email_code": "电子邮箱验证码错误.",
-        "incorrect_twofa_code": "手机2FA令牌双重验证码错误.",
-        "email_code_required": "需要电子邮件验证码,查看你的邮箱.",
-        "twofa_code_required": "需要手机2FA令牌双重验证."
+        "incorrect_login_pass": "用户名或密码错误。",
+        "incorrect_email_code": "电子邮箱验证码错误。",
+        "incorrect_twofa_code": "2FA两步验证码错误。",
+        "email_code_required": "需要电子邮件验证码，请检查您的邮箱。",
+        "twofa_code_required": "需要2FA两步验证码。"
```

Comment:
In Chinese usage, we don't typically use "2FA" as a standalone term. Instead, we say "令牌" or "2FA令牌," because the vast majority of ordinary users wouldn't understand what "2FA" means. "令牌" is a more straightforward and inclusive term, so I recommend using "需要2FA令牌双重验证" here.

"incorrect_twofa_code": "2FA令牌双重验证码错误。", "twofa_code_required": "需要2FA令牌双重验证。" "twofa_code": "2FA令牌验证码",